### ISSUE TITLE
Is it possible to migrate from zookeeper metadata store to etcd?

### ISSUE THREAD

**Issue by truong-hua** [apache/pulsar#15838]
### What issue do you find in Pulsar docs?

New 2.10.0 pulsar now supported ETCD as metadata store and we would like to migrate current zookeeper to etcd. Is it possible and is there any guidlines existed? Downtime should be acceptable :D

### What is your suggestion?

Provide metadatastore migration documents or a tool for doing it.

### Do you have any references?

Nope

### Would you like to fix this issue?

Nope

### Note

- [X] I have researched my question.

**Comment by Anonymitaet** [comment]
ping @momo-jun

**Comment by Anonymitaet** [comment]
Hi @momo-jun any feedback on this? Thanks!

**Comment by momo-jun** [comment]
Checked this issue with @nodece  -  it is not supported to migrate from ZK metadata store to ETCD.

**Comment by Anonymitaet** [comment]
@truong-hua truo
Thanks for raising this issue! Does the answer LGTY? Feel free to close this issue if you have more questions.

**Comment by nodece** [comment]
I suggest add documentation on how to switch the metadata store from ZK to ETCD/Memory/RockDB.

**Comment by truong-hua** [comment]
> I suggest add documentation on how to switch the metadata store from ZK to ETCD/Memory/RockDB.

Agree. Pulsar will be more portable and less depended on metadata store if we can do it.

**Comment by momo-jun** [comment]
> I suggest add documentation on how to switch the metadata store from ZK to ETCD/Memory/RockDB.

Make sense. @nodece can you provide more inputs?

**Comment by truong-hua** [comment]
@momo-jun is this document available yet?

**Comment by truong-hua** [comment]
@momo-jun there is a new doc to config etcd as metadata store but there is no document for migrating from a key-value store to another. So this issue is not closed yet.

**Comment by nodece** [comment]
@truong-hua Yes, the new doc is config etcd as metadatastore, right now we don't support migrating the data from one metadatastore to another.

**Comment by truong-hua** [comment]
So this issue should not be closed @momo-jun @nodece. Btw, if we can have a document about data structure or scheme of our current metadata on ZK and ETCD. May be people can think about migrating themselves manually, or may be some one can make a tool?

**Comment by truong-hua** [comment]
@tisonkun can this one be consider as to provide zookeeper metadata scheme structure doc and other like etcd. So the user can try themself to migrate it. I can not do it without deeply knowledge about the metadata scheme. A document about that would help.
